foundations/html_css/html-foundations/project-recipes.md

```diff
@@ -100,3 +100,17 @@ There are a couple of ways to go about doing this, but the simplest is this:
 -   Change the **Branch** from _none_ to _main branch_ and click **Save**. 
 -   It may take a few minutes (The GitHub website says up to 10, but we've seen it take up to an hour. Do not add a "theme" to your project, or you may have git conflicts, instead, be patient.) but your project should be accessible over the web from `your-github-username.github.io/your-github-repo-name` (obviously substituting your own details in the link).
 -   If your project does not publish after 1 hour, ensure that you have a file called `index.html` in the root of your repository and all the settings have been set correctly.  Go to your repo on GitHub and click on Actions, if there are no entries, then go back to the settings, change the **Branch** from _main branch_ to _none_ and click **Save**, then change the **Branch** from _none_ to _main branch_ and click **Save**.
+
+<div class="lesson-note" markdown="1">
+When you look at project submissions below, you might be asking:
+"Why do they look so good, should my project look like this?".
+
+Here is the answer:
+
+1- The main reason they look that good is that they are not done by beginners. They were likely submitted by people who had some experience with programming, or who have completed The Odin Project before and are coming back to make better websites.
+
+2- Your project should not look like that. Focus mainly on the requirements of the project.
+
+This applies for future projects too.
+For more information read Part 4 of [Becoming a TOP Success Story](https://dev.to/theodinproject/becoming-a-top-success-story-mindset-3dp2)
```

Comment:
Is this supposed to be Part 5 instead of Part 4?

Reply:
yes that's correct , maybe we should also change the link to be directly to the part, or provide both links

Reply:
Agreed!

Reply:
I made the required change